helm/crds/notification.kubesphere.io_dingtalkconfigs.yaml

```diff
@@ -38,97 +38,39 @@ spec:
           spec:
             description: DingTalkConfigSpec defines the desired state of DingTalkConfig
             properties:
-              chatbot:
-                description: Be careful, a ChatBot only can send 20 message per minute.
+              appkey:
+                description: The key of the application which sending message.
                 properties:
-                  keywords:
-                    description: Custom keywords of ChatBot
-                    items:
-                      type: string
-                    type: array
-                  secret:
-                    description: Secret of ChatBot, you can get it after enabled Additional
-                      Signature of ChatBot.
-                    properties:
-                      key:
-                        description: The key of the secret to select from.  Must be
-                          a valid secret key.
-                        type: string
-                      name:
-                        description: Name of the secret.
-                        type: string
-                      namespace:
-                        description: The namespace of the secret, default is the pod's
-                          namespace.
-                        type: string
-                    required:
-                    - key
-                    type: object
-                  webhook:
-                    description: The webhook of ChatBot which the message will send
-                      to.
-                    properties:
-                      key:
-                        description: The key of the secret to select from.  Must be
-                          a valid secret key.
-                        type: string
-                      name:
-                        description: Name of the secret.
-                        type: string
-                      namespace:
-                        description: The namespace of the secret, default is the pod's
-                          namespace.
-                        type: string
-                    required:
-                    - key
-                    type: object
+                  key:
+                    description: The key of the secret to select from.  Must be a
+                      valid secret key.
+                    type: string
+                  name:
+                    description: Name of the secret.
+                    type: string
+                  namespace:
+                    description: The namespace of the secret, default is the pod's
+                      namespace.
+                    type: string
                 required:
-                - webhook
+                - key
                 type: object
-              conversation:
-                description: The conversation which message will send to.
+              appsecret:
+                description: The secret of the application which sending message.
```

Comment:
The secret of the application which sending message. =>
The secret of the application with which to send messages.